packages/owlbot-bootstrapper/common-container/utils.ts

```diff
@@ -0,0 +1,101 @@
+import {execSync} from 'child_process';
+import {logger} from 'gcf-utils';
+import {ORG} from './common-container';
+import {uuid} from 'uuidv4';
+import {Octokit} from '@octokit/rest';
+import * as fs from 'fs';
+
+const BRANCH_NAME_FILE = 'branchName.md';
+export const REGENERATE_CHECKBOX_TEXT =
+  '- [x] Regenerate this pull request now.';
+const BRANCH_NAME_PREFIX = 'owlbot-bootstrapper-initial-PR';
+
+/**
+ * Saves the user name and email for owlbot-bootstrapper in git-credentials so as to not need to enter them when pushing using https protocol
+ */
+export async function setConfig(directoryPath: string) {
+  try {
+    execSync('git config --global user.name "Googleapis Bootstrapper"');
+    execSync(
+      'git config --global user.email "googleapis-bootstrapper[bot]@users.noreply.github.com"'
+    );
+    execSync(
+      `git config --global credential.helper 'store --file ${directoryPath}/.git-credentials'`
+    );
+  } catch (err) {
+    logger.error(err as any);
+    throw err;
+  }
+}
+
+/**
+ * Opens a new branch on a repo with a UUID, and saves that branch name to a well-known path
```

Comment:
In the comment, tell the reader the well-known path and tell them why the branch name is being written there.